configuration-sample/ods-core.env.sample

```diff
@@ -155,6 +155,7 @@ SHARED_LIBRARY_REPOSITORY=https://github.com/opendevstack/ods-jenkins-shared-lib
 # OpenShift #
 #############
 
+# Openshift v3.11.154
```

Comment:
Hmm is that the latest? I would note down maybe what is tested. But keep in mind that we need to constantly update this. I would avoid for OpenShift to be more specific than 3.11.

Reply:
You're right, I didn't notice the bugfix updates released every month or so. Makes sense that keeping at 3.11 is enough for now.